docs/rules/array-callback-return.md

```diff
@@ -26,8 +27,17 @@ This rule finds callback functions of the following methods, then checks usage o
 * [`Array.prototype.reduceRight`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.reduceright)
 * [`Array.prototype.some`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.some)
 * [`Array.prototype.sort`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.sort)
+* [`Array.prototype.forEach`](https://www.ecma-international.org/ecma-262/6.0/#sec-array.prototype.foreach)
```

Comment:
I think it's okay to have `forEach` in this list at the beginning of `Rule Details` although it isn't enabled by default, but it might be good to already note that this is optional. Also, the list is sorted.

Reply:
should it be put at the beginning of the list or in its sorted order (before map)?

Reply:
Hey I just updated the PR with this option in the list in its sorted position, I also added an observation regarding the `checkForEach` parameter, let me know if this is what you had in mind.

Reply:
Exactly that, thanks!